lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service.rb

```diff
@@ -92,6 +92,37 @@ def ansible_stats_vars
       MiqAeEngine::MiqAeAnsibleMethodBase.ansible_stats_from_hash(@persist_state_hash)
     end
 
+    def service_object
```

Comment:
Is this only meant for internal use?  If so it can be moved to the `private` section of the file.